ts/private/ts_project.bzl

```diff
@@ -19,12 +19,12 @@ def _gather_types_from_js_infos(targets):
     files_depsets.extend([
         target[JsInfo].transitive_types
         for target in targets
-        if JsInfo in target and hasattr(target[JsInfo], "transitive_types")
```

Comment:
Hmm, this looks like it was put here for backwards compatibility. Are we sure that it no longer an issue?

Reply:
rules_ts v3 (which `main` is for) only supports rules_js 2+ (and bazel-lib 2+) so I think this is ok

Reply:
okay, that makes sense